src/main/java/core/basesyntax/model/Message.java

```diff
@@ -1,8 +1,30 @@
 package core.basesyntax.model;
 
+import java.util.List;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+import org.hibernate.annotations.Cascade;
+import org.hibernate.annotations.CascadeType;
+
+@Entity
+@Table(name = "messages")
 public class Message {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String content;
+    @OneToMany(fetch = FetchType.EAGER)
+    @Cascade({CascadeType.PERSIST, CascadeType.DELETE})
```

Comment:
From checklist: "Be consistent, don’t use both session.save() and session.persist() in different daos. Choose single approach and use corresponding annotaions of Hibernate of JPA."

DELETE belongs to hibernate and REMOVE to JPA (and also for hibernate) 
change CascadeType.DELETE -> CascadeType.REMOVE